docs-2.0/4.deployment-and-installation/6.deploy-text-based-index/3.deploy-listener.md

```diff
@@ -1,68 +1,112 @@
 # Deploy Raft Listener for Nebula Storage service
 
-## Before you start
+Full-Text index data is written to the Elasticsearch cluster asynchronously. The Raft Listener (hereinafter shortened as Listener) is a separate process that receives requests from Nebula Graph clients and writes the data into the Elasticsearch cluster.
```

Comment:
It will not communicate with Graph side, just receive the wal from storage side, and then writes the data into ES cluster.